Dockerfile

```diff
@@ -23,9 +23,9 @@ RUN npm run build
 
 FROM ${BUILD_IMAGE} as runner
 
-RUN useradd nextjs
-RUN groupadd nodejs
-RUN usermod -a -G nodejs nextjs
+RUN useradd nextjs && \
```

Comment:
Not sure if it's useful but node images already ship with a non-root user https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#non-root-user

Reply:
We use our own node image